synapse/lib/stormtypes.py

```diff
@@ -4392,9 +4392,135 @@ class Str(Prim):
                        'desc': 'Keyword values which are substituted into the string.', },
                   ),
                   'returns': {'type': 'str', 'desc': 'The new string.', }}},
-        {'name': 'json', 'desc': 'Parse a JSON string and return the deserialized data.',
-         'type': {'type': 'function', '_funcname': '_methStrJson', 'args': (),
+         {'name': 'json', 'desc': 'Parse a JSON string and return the deserialized data.',
+          'type': {'type': 'function', '_funcname': '_methStrJson', 'args': (),
                   'returns': {'type': 'prim', 'desc': 'The JSON deserialized object.', }}},
+         {'name': 'isalnum', 'desc': '''
```

Comment:
The `name` values for these need to be updated to match the keys in `getObjLocals`.